go/vt/sqlannotation/sqlannotation.go

```diff
@@ -25,6 +27,10 @@ const (
 	deleteDML                            = "delete"
 )
 
+var (
+	replicationUnfriendlyCountStat = stats.NewInt("replication-unfriendly-statements-count")
```

Comment:
Sugu won't like this but I would go even further and name the variable ```filteredReplicationUnfriendlyCount``` (or ```...CountStat```) to be clear that this is not MySQL replication but filtered replication instead.
(You already have ```filteredReplicationUnfriendlyComment``` in the code :)

Just for reference for myself: Ammar is talking about the name of the stats variable, not the variable name itself.